data/data/powervs/cluster/bootstrap/variables.tf

```diff
@@ -0,0 +1,69 @@
+variable "memory" {
+  type = string
+  description = "The amount of memory to assign to each node in GB."
+}
+
+variable "processors" {
+  type = string
+  description = "The processor count for each node."
+}
+
+variable "ignition" {
+  type = string
+  description = "The ignition file."
+}
+
+variable "cloud_instance_id" {
+  type = string
+  description = "The Power VS Service Instance (aka Cloud Instance) ID."
+}
+
+variable "resource_group" {
+  type = string
+  description = "The name of the Power VS resource group to which the user belongs."
+}
+
+variable "image_id" {
+  type = string
+  description = "The ID of the Power VS boot image for the nodes."
+}
+
+variable "network_name" {
+  type = string
+  description = "The name of the Power VS network."
+}
+
+variable "proc_type" {
+  type = string
+  description = "The type of processor to be assigned (e.g. capped, dedicated, shared) to the nodes."
+}
+
+variable "sys_type" {
+  type = string
+  description = "The type of system on which to provision the nodes (e.g s922)."
+}
+
+variable "cluster_id" {
+  type = string
+  description = ""The ID created by the installer to uniquely identify the created cluster."
+}
+
+variable "key_id" {
+  type = string
+  description = "The SSH Key ID."
+}
+
+variable "cos_instance_location" {
+  type = string
+  description = "The region in which the Cloud Object Store resides. Used for the ignition file."
+}
+
+variable "cos_bucket_location" {
+  type = string
+  description = "The URL of the Cloud Object Store. Used for the igntion file."
```

Comment:
This is actually used for `region_location` [here](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/cos_bucket#region_location).

So it's not a URL, but a region like `us-south` or `eu-gb`.

Reply:
how would i differentiate that from `cos_instance_location` above?

Reply:
There isn't much difference, but `cos_instance_location` can be set to `global` unlike `region_location`. We actually use the `global` setting for `cos_instance_location` so as far as I can tell, we should keep both.